cookbook/case_studies/feature_engineering/feast_integration/feast_workflow.py

```diff
@@ -60,6 +63,23 @@
 DATA_CLASS = "surgical lesion"
 
 
+@task
+def create_bucket(bucket_name: str):
+    client = boto3.client(
+        's3',
+        aws_access_key_id=aws.S3_ACCESS_KEY_ID.get(),
+        aws_secret_access_key=aws.S3_SECRET_ACCESS_KEY.get(),
+        use_ssl=False,
+        endpoint_url="http://localhost:30084"
```

Comment:
Use https://github.com/flyteorg/flytesnacks/blob/master/cookbook/case_studies/feature_engineering/feast_integration/feast_dataobjects.py#L40 to configure the endpoint.